portfolio/src/main/webapp/js/loadMentees.js

```diff
@@ -1,6 +1,11 @@
-function loadMentees() {
+window.onload = function() {
+    this.loadMentors();
+}
+
+function loadMentors() {
     fetch('/load-mentees').then(response => response.json()).then((mentees) => {
```

Comment:
So I am a bit confused due to the variable names, but if I understand correctly then I think you want this file/function to load a list of mentors on the mentee page? If so, then I think this call to fetch() needs to have '/load-mentors' as the parameter. Also, maybe we can change the file names to "mentor.js" and "mentee.js" just so it's simpler and easier to associate them with the appropriate html files. Lmk what you think!

Reply:
Yeah you're correct, the variable names had me confused too so I'll change the file names to help